Yoast/wordpress-seo
# Improve vendor prefixing by only adding PHP files that aren't tests or examples

## Summary

This PR can be summarized in the following changelog entry:

* No user facing changes

## Relevant technical choices:

* This PR ensures only actual necessary PHP files are included in the `vendor_prefixed` directory. Documentation, tests and examples are excluded. `LICENSE` and `composer.json` files are still included.
* This reduces the `vendor_prefixed` directory size by about a third ( going from 1.9M to 1.3M ) which should aid in keeping ZIP sizes more manageable.

## Test instructions
This PR can be tested by following these steps:

* Run composer install
* Enable the `YOAST_SEO_INDEXABLES` feature flag.
* Uncomment the line `require_once WPSEO_PATH . 'src/loaders/oauth.php'; Temporarily disabled.` in `wp-seo-main.php`.
* Visit a variety of pages in the frondend.
* Create a new post in the backend.
* Visit the Yoast settings pages.
* No crashes should occur at any time.

## UI changes

## Quality assurance
